go-controller/pkg/cluster/master.go

```diff
@@ -74,6 +86,55 @@ func (cluster *OvnClusterController) StartClusterMaster(masterNodeName string) e
 	return nil
 }
 
+func (cluster *OvnClusterController) ReInitClusterNode() error {
+	// setup node
+	var err error
+	var subnet *netIPNet
+
+	nodes, err := cluster.Kube.GetNodes()
+	if err != nil {
+		logrus.Errorf("Error in getting k8s nodes: %v", err)
+		return err
+	}
+	for _, node := range nodes.Items {
+		sub, ok := node.Annotations[OvnHostSubnet]
+		_, subnet, err = net.ParseCIDR(sub)
+		if err != nil {
+			logrus.Errorf("Invalid hostsubnet found for node %s - %v",
+				node.Name, err)
+			return err
+		}
+
+		err = setupOVNNode(node.Name, config.Kubernetes.APIServer,
+			config.Kubernetes.Token)
+		if err != nil {
+			logrus.Errorf("Error setting up OVN node %s - %v", node.Name,
+				err)
+			return err
+		}
+		err = util.RestartOvnController()
```

Comment:
You can skip the call to RestartOvnController too. Since this again is local node dependent.